dhall/src/Dhall/Pretty/Internal.hs

```diff
@@ -1042,11 +1043,11 @@ prettyCharacterSet characterSet expression =
             <>  prettyExpression d
             <>  syntax rbrace
 
-        prettyText t = literal (Pretty.pretty (escapeText t))
+        prettyText t = literal (Pretty.pretty (escapeText_ t))
 
 -- | Pretty-print a value
-pretty :: Pretty a => a -> Text
-pretty = Pretty.renderStrict . Pretty.layoutPretty options . Pretty.pretty
+pretty_ :: Pretty a => a -> Text
```

Comment:
What's the motivation for this renaming?

Reply:
This is to work around a bug where haddocks are only rendered if the function is defined within the same module.  This is why `Dhall.Core` trivially re-exports this function, just to ensure that haddocks work